cloud/deploy/deploy.go

```diff
@@ -70,16 +70,11 @@ type deploymentInfo struct {
 }
 
 func deployDags(path, runtimeID string, client astro.Client) error {
-	dagDeployment, err := deployment.Initiate(runtimeID, client)
```

Comment:
Moving this below, since i want to do all validations before we request api for a sas link to upload